### DISCUSSION TITLE
how to customize Stacked Area ?

### DISCUSSION THREAD

**Discussion by ybf970928** [recharts/recharts#4048] [Q&A]
need make stacked Area charts,  when use same stackId, but the value displayed on the y-axis is the total, need to display the maximum value on the y-axis.
Example: 
<img width="697" alt="image" src="https://github.com/recharts/recharts/assets/63764959/d94adff8-a302-453d-aee6-fd56b33e7703">
<img width="689" alt="image" src="https://github.com/recharts/recharts/assets/63764959/5c32dd91-94c6-45ca-a893-01b5ac52f0a3">

**Comment by ybf970928** [comment]
like this , each Area is independent and will not cover each other
<img width="835" alt="image" src="https://github.com/recharts/recharts/assets/63764959/af8d0f73-d71d-47af-8913-bcf069762da7">

**Comment by ckifer** [comment]
This is how stacked area charts work, they add up to a total. Does having multiple, non stacked areas work for you?

You can set the YAxis domain to ["dataMin", "dataMax"] if you don't want to extend past your highest value.
